server/src/data/bots-data.js

```diff
@@ -20,6 +20,33 @@ class BotsData extends DataBase {
     return await this.BotModel.createOne(data);
   }
 
+  /**
+   * @param {object} options
+   * @param {string} [options.userId]
+   * @param {number} [options.offset]
+   * @param {number} [options.limit]
+   * @return {Promise<*|Array<Model>>}
+   */
+  async getAll(options) {
+    const pagination = {
+      offset: options.offset,
+      limit: options.limit,
+    };
```

Comment:
Do you think it would be reasonable to use the edges/nodes pagination instead?

Reply:
Yes, we should use only one way of pagination. Should I repair this right in the PR?

Reply:
Yea, that sounds good to me.

Reply:
Fixed.